src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -1443,7 +1460,7 @@ private void MergeHidingLookupResults(LookupResult resultHiding, LookupResult re
                             // SPEC: interface type, the base types of T are the base interfaces
                             // SPEC: of T and the class type object. 
 
-                            if (!IsDerivedType(baseType: hiddenContainer, derivedType: hidingContainer, basesBeingResolved, this.Compilation, useSiteInfo: ref useSiteInfo) &&
+                            if (!DerivesOrImplements(baseTypeOrImplementedInterface: hiddenContainer, derivedType: hidingContainer, basesBeingResolved, this.Compilation, useSiteInfo: ref useSiteInfo) &&
                                 !extendedTypeIsOrDerivedFrom(hiddenContainer, hidingContainer, basesBeingResolved, this.Compilation, ref useSiteInfo) &&
```

Comment:
>!extendedTypeIsOrDerivedFrom(hiddenContainer, hidingContainer, basesBeingResolved, this.Compilation, ref useSiteInfo) && [](http://example.com/codeflow?start=32&length=120)

Is this condition still necessary?

Reply:
Good catch. The only extensions scenarios still flowing through here now always involve an inheritance relationship, so the test is superfluous. Thanks